src/pages/index.js

```diff
@@ -122,17 +220,21 @@ window.addEventListener("resize", () => {
   profileDescriptionTooltipHandler.handleTooltip();
 });
 
+//profile picture modal
+profileImage.addEventListener("click", () => {
+  profilePicturePopup.open();
+});
+
 /* -------------------------------------------------------------------------- */
 /*                               Initialization                               */
 /* -------------------------------------------------------------------------- */
 
-//have initialCards render
-cardSection.renderItems();
-
-//enable form validation
-enableValidationOnAllForms(configObject);
-
-//create initial tooltips on page load
-cardTooltipHandler.handleTooltip();
-profileTitleTooltipHandler.handleTooltip();
-profileDescriptionTooltipHandler.handleTooltip();
+//have initialCards render and fetch userInfo from API, to set it in the profile
+Promise.all([initializeCards(), setUserInfoFromApi()]).then(() => {
+  //create initial tooltips on page load
+  cardTooltipHandler.handleTooltip();
+  profileTitleTooltipHandler.handleTooltip();
+  profileDescriptionTooltipHandler.handleTooltip();
+  //enable form validation
+  enableValidationOnAllForms(configObject);
+});
```

Comment:
Here you need `catch`

You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**